src/nxt_capability.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) Igor Sysoev
+ * Copyright (C) NGINX, Inc.
+ */
+
+#ifndef _NXT_CAPABILITY_INCLUDED_
+#define _NXT_CAPABILITY_INCLUDED_
+
+typedef struct {
+    uint8_t setid;
+} nxt_capability_t;
+
+NXT_EXPORT nxt_int_t nxt_capability_set(nxt_task_t *task, nxt_capability_t *cap);
+NXT_EXPORT void nxt_capability_log_hint(nxt_task_t *task);
+
+#if (NXT_HAVE_LINUX_CAPABILITY)
```

Comment:
The block below contains internals needed only for functions inside `src/nxt_capability.c`. It's better to move this block there instead of making it part of public `nxt_capability.h` interface.